docs/kafka.md

```diff
@@ -0,0 +1,50 @@
+# Kafka support in SCT
+
+## Requirements
+
+1) Be able to run a kafka setup, with a db cluster SCT created
+2) Have any of scylla connectors installed on it
+   1) from a tarball (local file / URL)
+   2) from confluent-hub (latest/specific version)
+3) Work locally with docker backend
+
+## Design
+
+![Design](./kafka-setup.png?raw=true )
+
+### Local setup
+
+we should be working with a https://github.com/conduktor/kafka-stack-docker-compose
+and it would run where the test is running (on the SCT runner)
+for case it won't be enough, we should support running it on it's own machine
+
+### High load setup
+
+for high load setups we can go in few routes
+* **SaaS** - confluent cloud / AWS MSK / others
```

Comment:
FWIW I vote for this route...

Reply:
Anyhow we want local development as the first stage.

Open an account for us in a confluent cloud, and we might check it and MSK to see which one fits better